datalad_dataverse/tests/test_utils.py

```diff
@@ -14,3 +46,22 @@ def test_format_doi():
         format_doi('')
     with pytest.raises(TypeError):
         format_doi(123)
+
+
+def test_dir_mangling_identity():
+    for p in _test_paths:
+        assert p == str(unmangle_directory_names(mangle_directory_names(p)))
+
+
+def test_file_mangling_identity():
+    for p in ["x/a", ".:*#?<>|;#"]:
```

Comment:
Minor comment: Guess, the escape character should be part of at least one of the tested names?

Reply:
Good point, thanks. Addressed in ebb65ac7924bff8c2fcf2411d71061e6d6561990